test/inputoutput/test_io.py

```diff
@@ -1,3 +1,13 @@
-from main.inputoutput.io import inc
+from main.inputoutput.io import inc,unzip_file
+import os
+
 def test_answer():
     assert inc(3) == 5
+
+def test_unzip_file():
+    fromPath=r"test\data\Desktop.zip"
```

Comment:
It is usually not a good practise to hardcode the path, as this will result in failures on Unix or Windows platform. 
Windows uses \ but Unix and other systems (like Mac) use /

Hence, tests are failing for me :)

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/3809547/54459353-5c25f480-475e-11e9-946f-f3d9a786f983.png">


You can use os.path.join() - See example [here](https://code-maven.com/slides/python-programming/os-path-join)